gunicorn/hatch.toml

```diff
@@ -1,13 +1,14 @@
 [env.collectors.datadog-checks]
 
 [[envs.default.matrix]]
-python = ["3.11"]
-version = ["19.9", "20.1"]
+python = ["3.12"]
+# v22 of gunicorn is the first that supports python 3.12
+version = ["22", "23"]
```

Comment:
wild, I thought we fixed this on master 🤯

Reply:
https://github.com/DataDog/integrations-core/blob/master/gunicorn/hatch.toml...interesting it doesn't fail. I can create a separate PR for this.